### PR TITLE
BaSP-TG-142: Landing Page

### PR DESCRIPTION
**Pull request by FiolAxel**
Created the landing page with JSX and CSS

### CODE REVIEW

File: src/components/Shared/Input/index.js [hunk 1]
```diff
@@ -1,10 +1,10 @@
 import styles from './input.module.css';
 
-function Input({ name, type, value, placeholder, error, onChange, min, max }) {
+function Input({ label = '', name, type, value, placeholder, error, onChange, min, max }) {
```
Comment (Gearyandres94) [inline src/components/Shared/Input/index.js:3]: Why do you add this props??

Comment (Gearyandres94) [inline src/components/Shared/Input/index.js:3]: Why do you add this props??

**Review by Gearyandres94 — CHANGES_REQUESTED** [review]

**Review by Gearyandres94 — APPROVED** [review]
🚀

**Review by Gearyandres94 — APPROVED** [review]

**Review by vadalamati — APPROVED** [review]
Goob job!

**Review by GuidoCerioni — APPROVED** [review]

**Pull request merged**